util/p9/clone

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+
+echo "cloning gatling-puppet-load-test"
+git clone https://github.com/puppetlabs/gatling-puppet-load-test.git
```

Comment:
Should this have a target destination path set?

Reply:
TDB whether a target destination path should be added. Added [docs/slv_environment_setup.md](https://github.com/puppetlabs/gatling-puppet-load-test/blob/SLV-321/docs/slv_environment_setup.md) with a [Utility scripts](https://github.com/puppetlabs/gatling-puppet-load-test/blob/SLV-321/docs/slv_environment_setup.md#utility-scripts) section.